apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -301,21 +319,33 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
         }
 
     elif result == "incorrect":
-        data["raw_submitted_answers"][name] = f"{random.randint(4, 100):d} * {a_tru}"
-        bigo_type = pl.get_enum_attrib(element, "type", BigOType, BIG_O_TYPE_DEFAULT)
-
-        if bigo_type is BigOType.THETA:
+        if a_tru == "":
```

Comment:
nit: Can you flatten this for readability as well into an `elif incorrect and a_tru == ""` and `elif incorrect`?